keyserver/coniksserver/main.go

```diff
@@ -45,10 +45,10 @@ func main() {
 
 	flag.Parse()
 
-	switch true {
+	switch {
 	case *genConfig:
 		mkConfig(*dir)
-		return
+		fallthrough
```

Comment:
I just tried with `coniksserver -genconfig` and it generated the config file only. So I think it worked as expected?